src/analysis/multiSpecies/Microbiome_Modeling_Toolbox/PairwiseInteractionModeling/createMultipleSpeciesModel.m

```diff
@@ -71,6 +74,18 @@
     mergeGenesFlag=false;
 end
     
+%% ensure compatibility with reconstructions from BIGG Models database
```

Comment:
I think we should rather handle this in `convertOldStyleModel` and not in an individual function.
I'll update the function soonish to handle these issues with BiGG models.